models/classes/LtiLaunchData.php

```diff
@@ -65,6 +65,15 @@ class LtiLaunchData implements \JsonSerializable
     public const LTI_SHOW_SCORE = 'custom_show_score';
     public const LTI_SHOW_CORRECT = 'custom_show_correct';
 
+    // for user claim
+    public const LTI_FOR_USER_ID = 'lti_for_user_id';
```

Comment:
Any reason which prevents reduced visibility to `private`?

Reply:
the same as other.
only as uniformity with other constants

Reply:
@boajer constant accessibility changed